### DISCUSSION TITLE
Orange C as a MSYS2 package

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#986] [Ideas]
MSYS2 integration: create a MSYS2 package for Orange C so I could install it with `pacman -S mingw-w64-x86_64-occ`. This will make it easier to use and test it with cmake when cmake support is available. MSYS2 is an excellent way to manage dependencies. Large projects always drag in a lot of dependencies, e.g: SDL2, GLFW,...

**Comment by GitMensch** [comment]
Sounds like a good idea. Please work on this (maybe using the CI definitions) and I'll help with the PKGBUILD file when you have an initial draft and questions.

**Comment by LADSoft** [comment]
this is a nice to have but I have other priorities right now...

**Reply by GitMensch** [reply]
Yes, that's a good idea - but it isn't rocket science and as @tqo50396 requested that I'd suggest that he/her/* creates the PGKBUILD recipe (and I'll help).

**Reply by ghost** [reply]
I proposed this idea since I want to use MSYS2 as the environment to test Orange C with cmake. MSYS2 offers a large number of ready-to-use libraries in the form of packages, and these libraries are compiled using GCC.

**Reply by ghost** [reply]
Do you ever think about compiling large software like wxWidgets or Qt with Orange C? If you want to stress-test your compiler, you should try. I think MSYS2 could help you with the dependencies, but from what you said on #974, I'm unsure now. Maybe you will have to compile all of the dependencies from source. I don't know.
